plasmapy/particles/atomic.py

```diff
@@ -663,14 +655,14 @@ def common_isotopes(
 
     See Also
     --------
-    ~plasmapy.utils.known_isotopes : returns a list of isotopes that
-        have been discovered.
+    `~plasmapy.particles.known_isotopes` : returns a list of isotopes
```

Comment:
```suggestion
    ~plasmapy.particles.known_isotopes : returns a list of isotopes
```

The "See Also" section is one of the weird situations where you do NOT use the back ticks `` ` ``.  If you do, `Sphinx` does not know how to interpret it and it shows up as `None()`.

Reply:
I just discovered that it's hard to find documentation for "See Also" in the Sphinx docs because all the "See Also" sections show up in the search results...

Reply:
Found [something about see also in the LSST docs](https://developer.lsst.io/python/numpydoc.html#py-docstring-see-also), and there was much rejoicing.